### PR TITLE
Update label and display of "Last gathered entries..." setting

### PR DESCRIPTION
**Pull request by marshmalien**
##### SUMMARY
- Update wording for AUTOMATION_ANALYTICS_LAST_ENTRIES label
- For readability, change "Last gathered entries..." detail as a code editor

Screen Shot 2021-12-13 at 2 56 54 PM

##### COMPONENT NAME

 - UI

### CODE REVIEW

**Comment by jlmitch5** [comment]
That looks perfect, thanks @marshmalien !

**Comment by InnocentK** [comment]
A few e2e tests around this context may require an update

**Review by nixocio — APPROVED** [review]
👍

File: awx/ui/src/screens/Setting/MiscSystem/MiscSystemDetail/MiscSystemDetail.js [hunk 1]
```diff
@@ -63,6 +67,10 @@ function MiscSystemDetail() {
       Object.keys(systemData).forEach((key) => {
         mergedData[key] = options[key];
         mergedData[key].value = systemData[key];
+
+        if (key === 'AUTOMATION_ANALYTICS_LAST_ENTRIES') {
+          mergedData[key].value = formatJson(systemData[key]) ?? '';
```
Comment (nixocio) [inline awx/ui/src/screens/Setting/MiscSystem/MiscSystemDetail/MiscSystemDetail.js:72]: @marshmalien good job. We could avoid re-write the value to mergedData.

```        mergedData[key].value =
          key === 'AUTOMATION_ANALYTICS_LAST_ENTRIES'
            ? (mergedData[key].value = formatJson(systemData[key]) ?? '')
            : systemData[key];
      });

```

But this is less readable IMO.

**Review by InnocentK — APPROVED** [review]
Looks good!

**Pull request merged**